src/Command/GenerateObjectAclCommand.php

```diff
@@ -103,6 +103,12 @@ public function execute(InputInterface $input, OutputInterface $output)
             }
         }
 
+        if (empty($this->aclObjectManipulators)) {
```

Comment:
Let's make this more explicit @phansys 

```suggestion
        if (null === $this->aclObjectManipulators) {
```

Reply:
I use empty function because `$this->aclObjectManipulators` is an array.
Using `null === $this->aclObjectManipulators` will not work